bricks/dart_frog_dev_server/hooks/pre_gen.dart

```diff
@@ -66,6 +67,28 @@ void reportRouteConflicts(
   }
 }
 
+void reportRogueRoutes(
+  HookContext context,
+  RouteConfiguration configuration,
+) {
+  if (configuration.rogueRoutes.isNotEmpty) {
+    context.logger.info('');
+    for (final route in configuration.rogueRoutes) {
+      final filePath = path.normalize(path.join('routes', route.path));
+      final fileDirectory = path.dirname(filePath);
+      final idealPath = path.join(
+        fileDirectory,
+        path.basenameWithoutExtension(filePath),
+        'index.dart',
+      );
+      context.logger.err(
```

Comment:
Shouldn't this be a warning? As it would still function without changing it?

Reply:
It won't function as expected similar to a route conflict. The rogue route will be unreachable unless the user moves it as per the message.